app/models/embassy.rb

```diff
@@ -5,7 +5,7 @@ def initialize(world_location)
 
   delegate :name, to: :@world_location
 
-  def consular_services_organisations
+  def organisations_with_embassy_offices
```

Comment:
Minor: Can you explain *why* you've renamed the method.

Reply:
I've now added additional explanation in the commit message and force-pushed.